packages/neon-core/src/rpc/RPCClient.ts

```diff
@@ -318,6 +318,30 @@ export class RPCClient {
     const response = await this.execute(Query.validateAddress(addr));
     return response.result.isvalid;
   }
+
+  /**
+   * Get the unspent utxo for an address
+   */
+  public async getUnspents(addr: string): Promise<any> {

```

Comment:
Can we strongly type these return values? I know the old RPC methods are not typed but lets start typing the new ones. It would be easier to work with them. For balance, you will need to pull in a parser (pretty close to the neoscan implementation).

Reply:
@snowypowers Good idea.
How about typing all RPC methods in neo3 version of neon-js?
For neo2, let's just keep it working like this.
What do you say?

Reply:
lets just implement it correctly here.

Reply:
@snowypowers OK. Done.